Formula/xcbeautify.rb

```diff
@@ -0,0 +1,21 @@
+class Xcbeautify < Formula
+  desc "Little beautifier tool for xcodebuild"
+  homepage "https://github.com/thii/xcbeautify"
+  url "https://github.com/thii/xcbeautify.git",
+      :tag      => "0.4.1",
+      :revision => "aa8b8795c9309c36d138bada74a73d1b8b16cda8"
+  head "https://github.com/thii/xcbeautify.git"
+  depends_on :xcode => ["10.2", :build]
+
+  def install
+    if MacOS.full_version < "10.14.4"
+      odie "macOS 10.14.4 or above is require to build xcbeautify."
+    end
+
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    system "#{bin}/xcbeautify", "--help"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.